### PR TITLE
Secrets cleanup

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:

- Secrets cleanup

**Related issue(s)**

### CODE REVIEW

File: docs/prow/prow-secrets.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+| **slack-token** | It stores oauth token for Slack bot user, it is used by Crier. |
+
+## workload-kyma-prow cluster
+
+| Secret   | Description | 
+| :---------- | :---------------- | 
+| **whitesource** keys | Are directly copied from the bucket when executing job. Secrets are not stored on the cluster. | 
+| **github-integration** | Are used to authorize Github applications configured in kyma-project organization (see `OAuth Apps` section in Github).|
+| **sa-*** | Service Accounts used in pipelines. You can find more information [here](/docs/prow/authorization.md).| 
+| **kyma-website-bot-zenhub-token** | Stores the ZenHub token for the `kyma-website-bot` account.|
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:27]:
```suggestion
| **kyma-website-bot-zenhub-token** | ZenHub token for the `kyma-website-bot` account.| 
```

File: docs/prow/prow-secrets.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+| **github-integration** | Are used to authorize Github applications configured in kyma-project organization (see `OAuth Apps` section in Github).|
+| **sa-*** | Service Accounts used in pipelines. You can find more information [here](/docs/prow/authorization.md).| 
+| **kyma-website-bot-zenhub-token** | Stores the ZenHub token for the `kyma-website-bot` account.| 
+| **kyma-bot-github-token**| Stores personal access token (called `Prow - Job`) used by the kyma-bot github user.| 
+| **kyma-guard-bot-github-token** | Stores personal access token of the kyma-guard-bot Github account.| 
+| **kyma-bot@sap.com**| Stores credentials to the kyma-bot Github account. |
+| **kyma-bot-npm-token** | It is a token for publishing npm packages in npmjs.com registry. Kyma-bot user credentials are used to authenticate to the registry. The secret is used by `post-master-varkes` ProwJob. |
+| **gardener-kyma-prow-kubeconfig** | It is a kubeconfig file, that allows connection to the Gardener `kyma-prow` project.| 
+| **slack-nightly-token**| It is a slack token that allows stability checker to push notifications to the Slack. |
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:33]:
```suggestion
| **slack-nightly-token**| Token that allows the stability checker to push notifications to Slack. | 
```

File: docs/prow/prow-secrets.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+| **sa-*** | Service Accounts used in pipelines. You can find more information [here](/docs/prow/authorization.md).| 
+| **kyma-website-bot-zenhub-token** | Stores the ZenHub token for the `kyma-website-bot` account.| 
+| **kyma-bot-github-token**| Stores personal access token (called `Prow - Job`) used by the kyma-bot github user.| 
+| **kyma-guard-bot-github-token** | Stores personal access token of the kyma-guard-bot Github account.| 
+| **kyma-bot@sap.com**| Stores credentials to the kyma-bot Github account. |
+| **kyma-bot-npm-token** | It is a token for publishing npm packages in npmjs.com registry. Kyma-bot user credentials are used to authenticate to the registry. The secret is used by `post-master-varkes` ProwJob. |
+| **gardener-kyma-prow-kubeconfig** | It is a kubeconfig file, that allows connection to the Gardener `kyma-prow` project.| 
+| **slack-nightly-token**| It is a slack token that allows stability checker to push notifications to the Slack. | 
+| **sap-slack-bot-token** | It is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).|
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:34]:
```suggestion
| **sap-slack-bot-token** | Token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot).|
```

File: docs/prow/prow-secrets.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+| **kyma-alerts-slack-api-url** | It is a token for publishing messages to the SAP CX workspace.  It is used by nightly and weekly ProwJobs.|
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:35]:
```suggestion
| **kyma-alerts-slack-api-url** | Token for publishing messages to the SAP CX workspace.  It is used by nightly and weekly ProwJobs.|
```

File: docs/prow/prow-secrets.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+| **neighbors-alerts-slack-api-url** | Publishes alerts to the private neighbors Slack channel.|
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:36]:
```suggestion
| **neighbors-alerts-slack-api-url** | Publishes alerts to the private `neighbors` Slack channel.|
```
Or maybe `...to the private Kyma Prow team Slack channel` or something like that? It's more informative and I think we shouldn't assume that the reader know Team Neighbors.

File: docs/prow/prow-secrets.md [hunk 6]
```diff
@@ -0,0 +1,39 @@
...
+| **kyma-azure-credential-*** | This set of secrets stores Azure subscription and service principal credentials. |
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:37]:
```suggestion
| **kyma-azure-credential-*** | Azure subscription and service principal credentials. |
```

File: docs/prow/prow-secrets.md [hunk 7]
```diff
@@ -0,0 +1,39 @@
...
+| **kyma-snyk-token** | It is token that allows authentication to Snyk CLI. It is used by `vulnerability-scanner` ProwJob. |
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:38]:
```suggestion
| **kyma-snyk-token** | Token that allows authentication to Snyk CLI. It is used by the `vulnerability-scanner` ProwJob. |
```

File: docs/prow/prow-secrets.md [hunk 8]
```diff
@@ -0,0 +1,39 @@
...
+| **kyma-website-bot-*** | Stores a personal access token of the kyma-website-bot Github account. It is responsible for publishing kyma-project.io website. |
```
Comment (majakurcius) [inline docs/prow/prow-secrets.md:39]:
```suggestion
| **kyma-website-bot-*** | Personal access token of the `kyma-website-bot` GitHub account. It is responsible for publishing the `kyma-project.io` website. |
```

File: docs/prow/prow-secrets.md [hunk 9]
```diff
@@ -0,0 +1,39 @@
...
+|**sap-slack-bot-token**| It is a token for publishing messages to the SAP CX workspace. Find more information [here](https://api.slack.com/docs/token-types#bot) |
+|**workload-cluster**| It stores `workload-kyma-prow` cluster certificate. |
+| **slack-token** | It stores oauth token for Slack bot user, it is used by Crier. |
+
+## workload-kyma-prow cluster
+
+| Secret   | Description | 
+| :---------- | :---------------- | 
+| **whitesource** keys | Are directly copied from the bucket when executing job. Secrets are not stored on the cluster. | 
+| **github-integration** | Are used to authorize Github applications configured in kyma-project organization (see `OAuth Apps` section in Github).|
```
[the remark this answers is not in the document]
Comment (kasiakepka) [reply docs/prow/prow-secrets.md:25]: I am referring to the `https://github.com/organizations/kyma-project/settings/applications` but when I placed this link governance job was [failing with 404](https://status.build.kyma-project.io/view/gcs/kyma-prow-logs/pr-logs/pull/kyma-project_test-infra/2259/pre-master-test-infra-governance/1252587469350637568).

But the link you placed is also good, thanks. :)

File: docs/prow/prow-secrets.md (hunk 5 above)
Comment (kasiakepka) [reply docs/prow/prow-secrets.md:36]: The name of the secret already refers to "neighbors" so I think we should leave it.

File: docs/prow/prow-secrets.md [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+## workload-kyma-prow cluster
+
+| Secret   | Description | 
+| :---------- | :---------------- | 
+| **whitesource** keys | Copied directly from the bucket when executing a job. These Secrets are not stored on the cluster. | 
+| **github-integration** | Used to authorize GitHub applications configured in the `kyma-project` organization. See the **[OAuth Apps](https://developer.github.com/apps/building-oauth-apps/)** section in GitHub.|
+| **sa-*** | Service Accounts used in pipelines. Find more information [here](/docs/prow/authorization.md).| 
+| **kyma-website-bot-zenhub-token** | ZenHub token for the `kyma-website-bot` account.| 
+| **kyma-bot-github-token**| Personal access token called `prow-job` used by the `kyma-bot` GitHub user.| 
+| **kyma-guard-bot-github-token** | Personal access token for the `kyma-guard-bot` GitHub account.|
```
Comment (dekiel) [inline docs/prow/prow-secrets.md:29]: This one is used by jobguard, we can change description to "Personal access token for the `kyma-guard-bot` GitHub account. It is used by jobguard."

File: docs/prow/prow-secrets.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (majakurcius) [reply docs/prow/prow-secrets.md]: I get the `404`, too, when I try to access the link you provided. 

Alrighty, no worries :)

**Pull request merged**